src/module.cpp

```diff
@@ -3220,6 +3141,12 @@ static void lSetPreprocessorOptions(const std::shared_ptr<clang::PreprocessorOpt
     opts->addMacroDef(ispc_major);
     opts->addMacroDef(ispc_minor);
 
+    char llvm_major[buf_size], llvm_minor[buf_size];
+    snprintf(llvm_major, buf_size, "LLVM_VERSION_MAJOR=%d", LLVM_VERSION_MAJOR);
+    snprintf(llvm_minor, buf_size, "LLVM_VERSION_MINOR=%d", LLVM_VERSION_MINOR);
+    opts->addMacroDef(llvm_major);
+    opts->addMacroDef(llvm_minor);
+
```

Comment:
It seems not to be related to the current commit and should be done separately. ispc.rst and ispc.vim should be updated accordingly.

Reply:
Ok, here is a separate PR https://github.com/ispc/ispc/pull/3120